Gopkg.toml

```diff
@@ -66,8 +66,8 @@
   version = "2.0.0"
 
 [[constraint]]
+  branch = "master"
```

Comment:
this PR is too huge for a single commit. 16K+ lines of code, please split it up.